deployments/cloud_security.yml

```
@@ -1238,6 +1238,18 @@ Resources:
             - Effect: Allow
               Action: dynamodb:*Item
               Resource: !Sub arn:${AWS::Partition}:dynamodb:${AWS::Region}:${AWS::AccountId}:table/panther-kv-store
+        - Id: GetPantherPolicySecrets
+          Version: 2012-10-17
+          Statement:
+            - Effect: Allow
+              Action: secretsmanager:GetSecretValue
+              Resource: !Sub arn:aws:secretsmanager:*:${AWS::AccountId}:secret:panther-policies/*
```

I don't think you can access secrets cross-region, in which case there's no reason not to interpolate the region here. Also we should add the partition to be standard with our other code.

```suggestion
              Resource: !Sub arn:${AWS::Partition}:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:panther-policies/*
```